docs/swagger/api.yaml

```diff
@@ -87,13 +87,28 @@ paths:
             items:
               $ref: '#/definitions/PrintReceiptResponse'
         400:
+          description:
+            Connection to the device is established,
+            but a problem with the device occured and receipt was not printed.
+            Returns array of warnings
```

Comment:
these wasning should be described like the PrintErrors in the schema. 
Swagger definitions are used for documentations and for code generations. There many different swagger code generators (https://github.com/swagger-api/swagger-codegen/tree/master/samples/server/petstore) that  can generate ready to use code for handling or calling the your service